docs/docs/modules/indexes/document_loaders/examples/file_loaders/directory.md

```diff
@@ -14,10 +14,17 @@ src/document_loaders/example_data/example/
 ├── example.json
 ├── example.jsonl
 ├── example.txt
-└── example.csv
+├── example.csv
+└── example.md
 ```
 
-Example code:
+## Setup
+
+```bash npm2yarn
```

Comment:
Remove the following instruction if decided to move `minimatch` from an optional to a required dependency.

Reply:
Done :)